tests/foreman/ui/test_host.py

```diff
@@ -2437,12 +2439,12 @@ def test_positive_host_registration_with_non_admin_user(
         default_organization=org,
         default_location=module_location,
     ).create()
-    # created_host = target_sat.api.Host(location=module_location, organization=org).create()
+    role = target_sat.cli.Role.info({'name': 'Register hosts'})
+    target_sat.cli.User.add_role({'id': user.id, 'role-id': role['id']})
+
     with Session(test_name, user=user.login, password=user_password) as session:
-        host = session.host.get_details(rhel8_contenthost.name, widget_names='breadcrumb')
-        assert host['breadcrumb'] == rhel8_contenthost.name
```

Comment:
why we removed this breadcrumb check?

Reply:
This was an unnecessary assertion which we are doing in multiple test cases. It just add on execution time.